packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -386,12 +387,13 @@ export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState
     }
 
     return (
-      <IconButton
+      <BaseButton
         tabIndex={ -1 }
         styles={ {
-          root: disabled ? styles.splitButtonMenuButtonDisabled : styles.splitButtonMenuButton,
-          rootChecked: styles.splitButtonMenuButtonChecked,
-          icon: disabled ? styles.splitButtonMenuIconDisabled : styles.splitButtonMenuIcon
+          root: disabled ? mergeStyles(styles.splitButtonMenuButton, styles.splitButtonMenuButtonDisabled) : styles.splitButtonMenuButton,
```

Comment:
I'd write these similar to how it is done in https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/Button/BaseButton.classNames.ts. Something like root: mergeStyles(styles.splitButtonMenuButtonDisabled, disabled &&.styles.splitButtonMenuButtonDisabled)